src/facade/dragonfly_connection.h

```diff
@@ -429,7 +425,12 @@ class Connection : public util::Connection {
   static thread_local QueueBackpressure tl_queue_backpressure_;
 
   // a flag indicating whether the client has turned on client tracking.
-  bool tracking_enabled_ = false;
+  bool tracking_enabled_ : 1 = false;
+  bool skip_next_squashing_ : 1 = false;  // Forcefully skip next squashing
+
+  // Connection migration vars, see RequestAsyncMigration() above.
+  bool migration_enabled_ : 1 = false;
+  bool is_http_ : 1 = false;
```

Comment:
> ../src/facade/dragonfly_connection.h:428:30: error: default member initializers for bit-fields only available with '-std=c++2a' or '-std=gnu++2a' [-Werror]
  428 |   bool tracking_enabled_ : 1 = false;

But why even? You saved 3 bytes per connection, there are lots of places where we waste memory more carelessly